src/axis.spec.js

```diff
@@ -120,8 +120,37 @@ describe('Axis', () => {
         expect(timeFormatDefaultLocaleSpy).toHaveBeenCalledWith(defaultLocale);
     });
 
+    it('should display tick using configuration locale', () => {
+        const data = [[{ id: 'foo' }]];
+        const selection = d3.select('svg').data(data);
+
+        let config = {
+            ...defaultConfig,
+            numberDisplayedTicks: { extra: 9 },
+        };
+
+        axis(d3, config, defaultScale)(selection);
```

Comment:
Use a `test` function to not repeat yourself and focus more on what you are testing.